cmd/sonobuoy/app/gen.go

```diff
@@ -133,6 +136,25 @@ func (g *genFlags) Config() (*client.GenConfig, error) {
 	}, nil
 }
 
+// resolveConformanceImage maps versions before 1.13 to Heptio's image and otherwise
+// to the upstream cnoformance image. Latest is always mapped to the upstream
+// regardless. The comparison is just lexical, e.g. "foo" < "v1.13" and "zip" >
+// "v1.13". These are a-typical and not given more support at this time.
+func resolveConformanceImage(imageVersion string) string {
+	// TODO(johnschnake): This logic should be temporary and is only
+	// required as we phase in the use of the upstream k8s kube-conformance
+	// image instead of our own heptio/kube-conformance one. They started
+	// publishing it for v1.13.
+	switch {
+	case imageVersion == ConformanceImageVersionLatest:
+		return config.UpstreamKubeConformanceImageURL
+	case imageVersion < "v1.13":
```

Comment:
Can `imageVersion` ever have an `alpha`, `beta`, or some other qualifier? If so, I believe this will fail the intended logic ([link](https://play.golang.org/p/_4tQj5Gv5sb)):

```go
package main

import (
	"fmt"
)

func main() {
	fmt.Println("1.13.alpha.0" < "1.13")
}
```

The above program prints `false` because when comparing lexicographically, `1.13.alpha.0` is greater than `1.13`.

Reply:
That shouldn't be a problem; the alpha and such always comes after the major.minor.patch. Upstream is good as far as I can tell about actually following the semver spec https://semver.org/#spec-item-9

If that were to ever happen I'd say it' a bug in the release flow.